esp/esp/tagdict/__init__.py

```diff
@@ -1,144 +1,943 @@
+from django import forms
+from django.forms import widgets
+from django.core.validators import RegexValidator
+from decimal import Decimal
+
+# Can't import this because of an import loop
```

Comment:
Instead of defining it twice, can the other place(s) import this one instead? This file feels like a "config" file, it makes sense if all other code that needs _states imports it from here.

Reply:
It feels kinda weird defining the `_states` list in the tagdict config file to be used elsewhere, but I guess it would certainly resolve the import issue.